File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_2/C/main.c
#include <stdio.h>

void parOuImpar(n1) {
  if (n1 % 2 == 0){
    printf("É par!");
  } else {
    printf("É impar!");
  }
};


int main() {
  double n1;

  printf("Digite um número: ");
  scanf("%lf", &n1);
  parOuImpar(n1);

  return 0;
}

File: LNPG_01_TIPOS_E_VARIAVEIS/ATIVIDADE_2/HASKELL/main.hs
parOuImpar :: Integer -> Bool
parOuImpar = even

main::IO()
main = do 
  putStrLn "Digite um numero: "
  input1 <- getLine
  let n1 = read input1 :: Integer
  if parOuImpar n1 then putStrLn "Par"
  else putStrLn "Impar"